app/scripts/views/tour/sidebars/economy-partial.html

```diff
@@ -0,0 +1,49 @@
+<h3 class="sidebar-content-heading mode-color">
+    <span class="header-underline">
+        Types of projects:
+    </span>
+</h3>
+<div class="sidebar-content-text">
+<ul>
```

Comment:
If you're referring to the giant html list, yeah this text will change for every view. I just copy/pasted the same list from energy so that something would be there. When we have actual content for each sidebar panel, we'll have to edit the content in each of these partials.